build/import/Packages.targets

```diff
@@ -2,41 +2,57 @@
+        Nerdbank.GitVersioning
+        Codecov
+      -->
+      https://api.nuget.org/v3/index.json;
     </RestoreSources>
   </PropertyGroup>
 
   <ItemGroup>
     <!-- Infrastructure -->
     <PackageReference Update="MicroBuild.Core.Sentinel"                                               Version="1.0.0" />
     <PackageReference Update="Microsoft.VisualStudioEng.MicroBuild.Core"                              Version="1.0.0" />
-
-    <!-- Toolset -->
+    <PackageReference Update="Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild"                 Version="1.1.286" />
     <PackageReference Update="Nerdbank.Streams"                                                       Version="2.8.57" />
+    <PackageReference Update="Nerdbank.GitVersioning"                                                 Version="3.5.107" />
     <PackageReference Update="System.IO.Pipelines"                                                    Version="6.0.1" />
-    <PackageReference Update="RoslynTools.RepoToolset"                                                Version="$(RoslynToolsRepoToolsetVersion)" />
-    <PackageReference Update="RoslynTools.ModifyVsixManifest"                                         Version="$(RoslynToolsModifyVsixManifestVersion)" />
-    <PackageReference Update="RoslynTools.SignTool"                                                   Version="$(RoslynToolsSignToolVersion)" />
     <PackageReference Update="XliffTasks"                                                             Version="1.0.0-beta.20574.1" />
-    <PackageReference Update="Microsoft.DiaSymReader.Pdb2Pdb"                                         Version="$(MicrosoftDiaSymReaderPdb2PdbVersion)" />
+    <!-- This is used for publishing PDBs to the legacy symbol server. It converts portable PDBs to Windows PDBs (embedded). -->
+    <!-- https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/672/Archive-Symbols-with-Symweb?anchor=portable-pdbs -->
+    <PackageReference Update="Microsoft.DiaSymReader.Pdb2Pdb"                                         Version="1.1.0-beta2-22320-02" />
     <PackageReference Update="Microsoft.Net.Compilers.Toolset"                                        Version="4.2.0-1.final" />
-    <PackageReference Update="OpenCover"                                                              Version="4.7.1138-rc" />
-    <PackageReference Update="Codecov"                                                                Version="$(CodecovVersion)" />
-    <PackageReference Update="MicroBuild.Plugins.SwixBuild"                                           Version="$(MicroBuildPluginsSwixBuildVersion)" />
-    <PackageReference Update="Microsoft.DevDiv.Validation.MediaRecorder"                              Version="15.0.199" />
+    <!-- This package is deprecated. CodecovUploader is now the recommended package. -->
+    <PackageReference Update="Codecov"                                                                Version="1.13.0" />
```

Comment:
This wasn't changed as this issue outlines be the better fix: https://github.com/dotnet/project-system/issues/8219